### ISSUE TITLE
It doesnt work for me? 

### ISSUE THREAD

**Issue by Sas0ni** [fredemmott/StreamDeck-Discord#76]
I do everything correct like in this video https://www.youtube.com/watch?v=MSMbRtj2fFA , but still dont work?

**Comment by Sas0ni** [comment]
Like it doesnt pop up in discord like it should

**Comment by Aeswere** [comment]
Yeah it doesn't work for me either after it previously did. Think it just stopped today, debug info on the actions say requesting user permission

**Comment by Sas0ni** [comment]
:/

**Comment by Aeswere** [comment]
Well I'm not sure why mine had stopped working but I did fix it.

When you make an application, you'll need the `Client ID` & `Client Secret` from the `OAuth2` tab in the [discord applications developer area](https://discord.com/developers/applications), fill in those values in the places where the action asks for it. Also in the OAuth2 area, on the dev page, click `Add Redirect` and enter in `https://localhost/` and save changes. It *should* work if you then click reauthenticate.

**Comment by schmittyd** [comment]
I have been having issues for the last couple of months too. Pressing Reauthenticate or Reconnect does not do anything.

**Comment by Aeswere** [comment]
> I have been having issues for the last couple of months too. Pressing Reauthenticate or Reconnect does not do anything.

Have you tried reinstalling? and redoing [the stuff I mentioned in my comment?](https://github.com/fredemmott/StreamDeck-Discord/issues/76#issuecomment-847234232) The only times I've had issues, clicking reauth fixes it for me.
